source/lex.tex

```diff
@@ -1177,6 +1184,18 @@
   \tcode{unsigned long long int}  &
   \tcode{unsigned long long int}\\
 and \tcode{ll} or \tcode{LL}  &
+                              &
+                              \\\hline
+\tcode{z} or \tcode{Z}    &
+  the signed integer type &
+  the signed integer type \\
+                          &
+  corresponding to \tcode{std::size_t}\iref{support.types.layout} &
+  corresponding to \tcode{std::size_t} \\\hline
```

Comment:
@jensmaurer: can you double-check, is it an accident that you omitted the plain `std::size_t` in the binary/octal/hex column?

(I can fix this during merge, no need to send any edits.)

Reply:
Pushed a fix. Thanks for noticing.